anthos-bm-openstack-terraform/README.md

```diff
@@ -1,13 +1,43 @@
-# Terraform example to create OpenStack VMs for Anthos
-This repository contains an example of what OpenStack resources you need to
-create for setting up Anthos on Bare Metal. The following resources are
-created as part of the Terraform module:
+## Anthos Baremetal on OpenStack with Terraform
```

Comment:
Both "baremetal" and "bare metal" are used in this PR -- is there a preference for one or the other?

Reply:
Should be space separated. Fixed all of it!